core/src/main/java/xtdb/trie/LeafMergeQueue.java

```diff
@@ -35,31 +36,44 @@ public String toString() {
 
     private final byte[] path;
     private final IVectorReader[] rdrs;
+    private final HashMap<Integer, Integer> ordinalToIndex;
     private final PriorityQueue<LeafPointer> pq;
```

Comment:
Realized during Standup that we could also just maintain a map from <ordinal, ivectorreader>.

Reply:
Addressed.